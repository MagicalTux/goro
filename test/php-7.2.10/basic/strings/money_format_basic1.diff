     *** Testing money_format() : basic functionality***
     Format values with 14 positions, 8 digits to left, 2 to right using national format
003- string
004- string
005- Format again but with ( for negative values
006- string
007- string
008- Format with 0 for padding character
009- string
010- string
011- Format again with * for padding character
012- string
013- string
014- Format again but disable grouping character
015- string
016- string
017- Format again suppress currency symbol
018- string
019- string
020- ===DONE===
003+ 
004+ Fatal error: Uncaught Error: Call to undefined function money_format in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php:20
005+ Stack trace:
006+ #0 {main}
007+   thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php on line 20
