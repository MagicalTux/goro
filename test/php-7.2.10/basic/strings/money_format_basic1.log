
---- EXPECTED OUTPUT
*** Testing money_format() : basic functionality***
Format values with 14 positions, 8 digits to left, 2 to right using national format
string
string
Format again but with ( for negative values
string
string
Format with 0 for padding character
string
string
Format again with * for padding character
string
string
Format again but disable grouping character
string
string
Format again suppress currency symbol
string
string
===DONE===
---- ACTUAL OUTPUT
*** Testing money_format() : basic functionality***
Format values with 14 positions, 8 digits to left, 2 to right using national format

Fatal error: Uncaught Error: Call to undefined function money_format in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php:20
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php on line 20
---- FAILED
