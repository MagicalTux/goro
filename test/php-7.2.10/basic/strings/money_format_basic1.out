*** Testing money_format() : basic functionality***
Format values with 14 positions, 8 digits to left, 2 to right using national format

Fatal error: Uncaught Error: Call to undefined function money_format in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php:20
Stack trace:
#0 {main}
  thrown in /home/ronald/proj/goro/test/php-7.2.10/basic/strings/money_format_basic1.php on line 20